version.txt

```diff
@@ -1 +1 @@
-0.4.8.20241014
+0.4.8.20241015
```

Comment:
@tyler92, can you bump the patch version? You've changed the internal implementation without changing any public interfaces, so that should normally be done.

Reply:
Done, 0.4.10.20241017